La till actionbars och en loginActivity

willedanielsson opened the pull request:
La till ikoner för framtid bruk samt att jag fixade actionbar för fragmenten Map och Case som har ikoner som man kan trycka på. För case så gjorde jag addCase i actionbaren vilket betyder att den stora knappen nu ligger lite fint i actionbaren. I Map gjorde jag två actionknappar, addObstascle och visualSettings. Det första som hände rnu i appen är att man kommer till en Login-fas. Om man trycker på logga in så hamnar man då i Ärende

willedanielsson committed:
Samma kod som Henning fast jag har lagt in massa ikoner för framtid bruk samt att nu finns det en actionbar till Map och Case som innehåller ikoner som fungerar som knappar, dvs att knappen som fanns i Case inte längre finns där

willedanielsson committed:
Fixade så att knappen addCasen i actionbaren tar dig till addCaseFragment.java

willedanielsson committed:
Merge branch 'master' of https://github.com/Rovanion/AinAppen

Conflicts:
	res/drawable-hdpi/ic_drawer.png
	res/drawable-mdpi/drawer_shadow.9.png
	res/drawable-mdpi/ic_drawer.png
	res/drawable-xhdpi/drawer_shadow.9.png
	res/layout/activity_contacts.xml
	res/layout/drawer_list_item.xml
	res/layout/fragment_case.xml
	res/layout/fragment_main.xml
	res/layout/fragment_map.xml
	res/values/strings.xml
	src/gov/polisen/ainappen/CaseFragment.java
	src/gov/polisen/ainappen/MapFragment.java

willedanielsson committed:
La till login_activity. Nu kommer vi att starta med en login-ruta och om man trycker på logga in så hamnar man på mainactivity

willedanielsson committed:
Tog bort onödiga filer

willedanielsson committed:
Tog bort ännu mer onödiga filer

henninghall commented:
Fungerar bra! Saker som kan men inte måste fixas:
- Bilderna är samma upplösningar i alla bildmappar -> fel
- Action bar buttons borde döljas när navigation drawer öppnas.

Rovanion commented:
I framtiden kan du komma ihåg att köra $ git commit --amend  om du kom på någonting som du ville haft med i förra commiten, t.ex. som att ta bort lite fler onödiga filer.

Rovanion commented on src/gov/polisen/ainappen/CaseFragment.java:
```diff
@@ -18,8 +21,32 @@ public CaseFragment() {
+	public boolean onOptionsItemSelected(MenuItem item) {
+	   // handle item selection
+	   switch (item.getItemId()) {
+//If we press the addCasebutton in the actionbar, call the addCase function in MainActivity	   
```
Den här kommentaren är inte indenterad korrekt.

Rovanion replied:
http://en.wikipedia.org/wiki/Indenting#Indentation_in_programming

Kommentaren är inte i höjd med resten av koden.

willedanielsson replied:
Ändrar genom att ta bort massa bilder för högre resolution, dem var där för att fylla det tomrummet och jag ändrade inte heller upplösningen bara för att det tar onödig tid.

Angående Hennings kommentar så ska verkligen inte actionbarButtons försvinna när drawern öppnas, det känns jätte konstigt

Rovanion commented on src/gov/polisen/ainappen/LoginAcitivity.java:
```diff
@@ -0,0 +1,51 @@
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.os.Build;
```
Här i imports så finns det importer som inte används.

Rovanion commented on src/gov/polisen/ainappen/MapFragment.java:
```diff
@@ -15,7 +18,32 @@ public MapFragment() {
+		public boolean onOptionsItemSelected(MenuItem item) {
+		   // handle item selection
+		   switch (item.getItemId()) {
+	//If we press the addObstacle in the actionbar, call the addObstacle function in MainActivity	   
```
Å så även den här.

Rovanion commented:
Ett större problem med ikonerna är att dom för dom olika dpi:erna är samma upplösning. Och att vissa av ikonerna är enorma gör inte saken bättre.

willedanielsson committed:
Ändrar genom att ta bort massa bilder för högre resolution, dem var där för att fylla det tomrummet och jag ändrade inte heller upplösningen bara för att det tar onödig tid.

Angående Hennings kommentar så ska verkligen inte actionbarButtons försvinna när drawern öppnas, det känns jätte konstigt

henninghall commented:
Det känns inte alls jättekonstigt, tvärt om. Problem med actionbar button som inte försvinner när man öppnar drawern:
- Emot guidlines
- Man öppnar drawern för att byta sida, alltså vill man inte göra saker med nuvarande.
- Om man har drawer öppen, och använder (Skapa nytt ärande) i action bar, öppnas den i bakgrunden och drawer är fortfarande öppen - man ser inte vad man gör --> fail.
- Gmail-appen (som vi använder som förebild) döljer dessa när drawern öppnas. Testa gärna. 

Jag kan dock fixa det om du vill wille, men tror det är lättast om du gör det själv :)

willedanielsson commented:
Okej det kommer till nästa update, inte denna! :)

Rovanion commented:
När jag merge:a jakbas pull request så gick den i konflikt med den här. Jag kan inte koden så vi får manuellt merge:a den här imorn.
